File: Fundamentals C#/Basic Syntax, Cond State, Loops/Basic Syntax, Conditional Statements and Loops - Exercise/04. Print and Sum.cs
// See https://aka.ms/new-console-template for more information




int start = int.Parse(Console.ReadLine());
int stop = int.Parse(Console.ReadLine());
int sum = 0;
for (int i = start; i <= stop; i++)
{
    Console.Write($"{i} ");
    sum += i;
    if(i==stop)
        Console.WriteLine();
}
Console.WriteLine($"Sum: {sum}");

File: JavaScript Front-End/02.Syntax Fundamentals Exercise/04. Print And Sum.js
function solve(start, end){
    let numbers = "";
    let sum = 0;
    for (let i = start; i <= end; i++) {
        numbers += `${i} `  
        sum += i;
    }

    console.log(numbers);
    console.log(`Sum: ${sum}`);
}
solve(5, 10);